meinberlin/apps/plans/migrations/0062_alter_plan_description.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.2.9 on 2024-01-30 09:17
+
+import adhocracy4.images.validators
+from django.db import migrations
+import django_ckeditor_5.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("meinberlin_plans", "0061_rename_m2mtopics_plan_topics"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="plan",
+            name="description",
+            field=django_ckeditor_5.fields.CKEditor5Field(
+                help_text="Describe the key points of your plan. You can upload PDFs and images, embed videos and link to external URLs, among other things.",
+                validators=[adhocracy4.images.validators.ImageAltTextValidator()],
```

Comment:
here too, move help_text in PlanForm.
import validator class at the top.

Reply:
is done in #5412 already